pkg/model/manifest.go

```diff
@@ -559,6 +559,15 @@ func GetInferredManifest(cwd string) (*Manifest, error) {
 					},
 				},
 			},
+
+			Destroy: &DestroyInfo{
+				Commands: []DeployCommand{
+					{
+						Name:    fmt.Sprintf("kubectl delete -f %s", manifestPath),
```

Comment:
@jLopezbarb what should be the expected behavior on `destroy` if there is no okteto manifest?

Reply:
If the Okteto manifest isn't there, it'll look at whether there are other manifests such as Kubernetes, helm, and docker-compose. If none exist, it'll panic like in the issue this PR is trying to solve. 

Even if there are manifests other than okteto manifest existing, we've only provided commands in the `manifest.go` on how to deploy (like _kubectl apply -f ..._) but not provided commands on how to destroy these manifests if no okteto manifest is there.

Reply:
> @jLopezbarb what should be the expected behavior on `destroy` if there is no okteto manifest?

If the name is passed or we can infer it from one of the manifest we should destroy that environment